pluginMeasureCpu.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, ts: TestSettings, task_role: TaskRole, tenant: bool):
         self.out_file_yaml = tftbase.get_manifest_renderpath(
             f"tools-pod-{self.node_name}-measure-cpu.yaml"
```

Comment:
If you give the pod another name, then the yaml file should also follow (and use the sanitized name).